ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -99,30 +99,32 @@ function ProfileChecksPage() {
         history.push(path);
     }
 
+    const selectedProfileDetails = scanConfigProfilesResponse?.profiles.find(
+        (profile) => profile.name === profileName
+    );
+
     return (
         <>
             <PageTitle title="Compliance coverage - Profile checks" />
             <CoveragesPageHeader />
-            <PageSection>
-                <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsMd' }}>
-                    {!isDisclaimerAccepted && (
-                        <ComplianceUsageDisclaimer onAccept={() => setIsDisclaimerAccepted(true)} />
-                    )}
-                    <ProfilesToggleGroup
-                        profiles={profileScanStats.scanStats}
-                        handleToggleChange={handleProfilesToggleChange}
-                    />
-                </Flex>
-            </PageSection>
-            <PageSection variant="default" className="pf-v5-u-py-0">
+            {!isDisclaimerAccepted && (
+                <ComplianceUsageDisclaimer onAccept={() => setIsDisclaimerAccepted(true)} />
```

Comment:
Brad, to verify visual effect of layout simplification, can you include a picture with disclaimer?